pyscriptjs/src/runtime.ts

```diff
@@ -177,7 +177,7 @@ export abstract class Runtime extends Object {
 
         loader?.log('Initializing scripts...');
         for (const script of scriptsQueue_) {
-            await script.evaluate();
+            void script.evaluate();
```

Comment:
this shows my ignorance of JS: what does `void` means in this context? Is it any different than just doing `script.evaluate()`?

Reply:
script.evaluate() returns a Promise and you can handle that 2 ways:

1. using .catch or .then to process Promise results. 
```typescript
script.evaluate().catch(console.log)
```
This will trap errors in the Promise execution and passes error result to console.log

2. Voiding the result as it just contains nothing of value. Here the Promise just returns when the pyscript completes the evaluation. In effect the Promise just says, I am done. In the case of an endless loop, nothing returns.
```typescript
void script.evaluate()
```
This just prevents the Promise from being assigned to anything and Typescript is also happy. `const foo = void 1` ==> foo === undefined

Reply:
ah ok, so it's basically just an annotation to make the typescript compiler happy. Thanks!